src/session/DailyEventHandler.ts

```diff
@@ -66,13 +66,22 @@ export class DailyEventHandler {
 
     const stream = createStream(participant);
 
+    // session connected
+
     this.ee.emit("streamCreated", getStreamCreatedEvent(this.session, stream));
     this.ee.emit(
       "connectionCreated",
       getConnectionCreatedEvent(this.session, connection)
     );
   }
 
+  // onLocalParticipantJoined() handles Daily's "participant-updated" event for
+  // local participant
+  onLocalParticipantUpdated(participant: DailyParticipant) {
+    const stream = createStream(participant);
```

Comment:
I suggest maybe adding some info to the comment above about how this is really intended to be called once. Otherwise it might be confusing, since the reader might think "Do we really want to send a stream created event on every single participant update?" (which is what I thought until I read the part in `Session.ts` below :D )